core/help/help.py

```diff
@@ -24,6 +25,23 @@
 )
 
 ctx = Context()
+
+def update_lists():
+    mapping = {}
+    for name in registry.lists.keys():
+        list_name = name.split('.')[-1]
+        display_name = list_name.replace("_", " ")
+
+        short_names = actions.user.create_spoken_forms(
+            display_name,
+            generate_subsequences=False,
+        )
+        for short_name in short_names:
+            mapping[short_name] = list_name
+
+    ctx.lists["user.help_lists"] = mapping
+
+update_lists()
```

Comment:
I'm not sure if doing this globally is the right way to do it, as I don't see that being done for `user.help_contexts`.